dts/msm8916/msm8916-mtp.dts

```diff
@@ -7,6 +7,12 @@
 / {
 	compatible = "qcom,msm8916-mtp", "qcom,msm8916";
 
+	qcom,msm-id = <206 0>, <248 0>, <249 0>, <250 0>; // Provide a set of msm-id for passing dtbTool
+	qcom,board-id = <8 0 15005>, /* OPPO board-id hax */
```

Comment:
Please try `qcom,board-id = <8 0>`, without the OPPO numbers.

Reply:
Yeah, I was thinking about that, that's why I asked you guys to test.

Reply:
Yeah it actually works fine, nice to see that.